cli/src/pcluster/config/update_policy.py

```diff
@@ -679,7 +696,7 @@ def condition_checker_login_nodes_stop_policy(_, patch):
     action_needed=UpdatePolicy.ACTIONS_NEEDED["login_nodes_stop"],
 )
 
-# Update supported only with all login nodes down
+# Update supported only with all login nodes in the cluster down
 UpdatePolicy.LOGIN_NODES_STOP = UpdatePolicy(
```

Comment:
why is this policy needed if we have LOGIN_NODES_POOL_STOP? Is there any update policy for changes to a pool, such as increasing the number of nodes in a pool?

Reply:
It's still used [here](https://github.com/aws/aws-parallelcluster/blob/develop/cli/src/pcluster/schemas/cluster_schema.py#L1922) to update the entire login nodes section 

The update policy for changes to a pool is now LOGIN_NODES_POOL_STOP